spec/02-integration/03-db/11-postgres-ro_spec.lua

```diff
@@ -144,9 +144,8 @@ for _, strategy in helpers.each_strategy() do
 
         helpers.wait_until(function ()
           return pcall(function ()
-            assert.logfile().has.line("get_updated_router(): could not rebuild router: " ..
-                                      "could not load routes: [postgres] connection " ..
-                                      "refused (stale router will be used)", true)
+            assert.logfile().has.line("could not rebuild router.*: could not load routes.*" ..
+                                      "postgres.*connection refused", false)
```

Comment:
this change seems related to the new atc-router. now, we don't get exactly the same error about the RO pg error. we are not getting:
https://github.com/Kong/kong/blob/master/kong/runloop/handler.lua#L1125
instead of:
https://github.com/Kong/kong/blob/master/kong/runloop/handler.lua#L838


@dndx could just double check if this is expected?